modules/nf-core/gatk4/germlinecnvcaller/main.nf

```diff
@@ -0,0 +1,76 @@
+process GATK4_GERMLINECNVCALLER {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'broadinstitute/gatk:4.4.0.0':
+        'broadinstitute/gatk:4.4.0.0' }"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "GATK4_GERMLINECNVCALLER module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    input:
+    tuple val(meta), path(tsv)
+    path intervals
+    path model
+    path ploidy
+
+    output:
+    tuple val(meta), path("*-calls.tar.gz"), emit: calls, optional: true
+    tuple val(meta), path("*-model.tar.gz") , emit: model, optional: true
+    path  "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def intervals_command = intervals ? "--intervals $intervals" : ""
+    def untar_ploidy = ploidy ? (ploidy ==~ /^.*\.tar\.gz$/ ? "tar -xzf ${ploidy}" : "") : ""
+    def untar_model = model ? (model ==~ /^.*\.tar\.gz$/ ? "tar -xzf ${model}" : "") : ""
+    def ploidy_command = ploidy ? (ploidy ==~ /^.*\.tar\.gz$/ ? "--contig-ploidy-calls ${ploidy.toString().replace(".tar.gz","")}" : "--contig-ploidy-calls ${ploidy}") : ""
+    def model_command = model ? (model ==~ /^.*\.tar\.gz$/ ? "--model ${model.toString().replace(".tar.gz","")}/${prefix}-model" : "--model ${model}/${prefix}-model") : ""
+    def input_list = tsv.collect{"--input $it"}.join(' ')
+    def output_command = model ? "--output ${prefix}-calls" : "--output ${prefix}-model"
+    def tar_output = model ? "tar -czf ${prefix}-calls.tar.gz ${prefix}-calls" : "tar -czf ${prefix}-model.tar.gz ${prefix}-model"
+
+    def avail_mem = 3
```

Comment:
Try and use the same syntax as used for the other gatk modules. Example
https://github.com/nf-core/modules/blob/880d634b0d5aead9447ae29e3e02e0e31ca7ae7f/modules/nf-core/gatk4/bedtointervallist/main.nf#L25